matrix.yaml

```diff
@@ -139,3 +160,15 @@ exclusions:
   # Java 8 is not needed for Lucee 6.0
   - TOMCAT_JAVA_VERSION: 'jre8-temurin-jammy'
     LUCEE_MINOR: '6.0'
+  - TOMCAT_JAVA_VERSION: 'jdk8-temurin-jammy'
+    LUCEE_MINOR: '6.0'
+  - TOMCAT_JAVA_VERSION: 'jre8-temurin-jammy'
+    LUCEE_MINOR: '6.1'
+  - TOMCAT_JAVA_VERSION: 'jdk8-temurin-jammy'
+    LUCEE_MINOR: '6.1'
+  - TOMCAT_JAVA_VERSION: 'jre21-temurin-jammy'
+    LUCEE_MINOR: '5.3'
```

Comment:
why?

Reply:
@zspitzer we do not Java 8 build for Lucee 6 and no java 21 builds for Lucee 5